### PR TITLE
CONFIGS: Make gfx presets consistent

### PR DESCRIPTION
**Pull request by drzel**

### CODE REVIEW

File: misc/cfg/gfx_gl_eyecandy.cfg [hunk 1]
```diff
@@ -116,15 +121,14 @@ gl_miptexLevel               "0"
 gl_picmip                    "0"
 gl_playermip                 "0"
 gl_scaleModelTextures        "0"
-gl_scaleTurbTextures         "0"
+gl_scaleTurbTextures         "1"
 gl_subdivide_size            "64"
-gl_textureless                 "0"
-gl_texturemode               "gl_linear_mipmap_linear"
+gl_textureless               "0"
+gl_texturemode               "GL_LINEAR_MIPMAP_NEAREST"
```
Comment (meag) [inline misc/cfg/gfx_gl_eyecandy.cfg]: gl_linear_mipmap_linear

File: misc/cfg/gfx_gl_eyecandy.cfg [hunk 2]
```diff
@@ -116,15 +121,14 @@ gl_miptexLevel               "0"
 gl_picmip                    "0"
 gl_playermip                 "0"
 gl_scaleModelTextures        "0"
-gl_scaleTurbTextures         "0"
+gl_scaleTurbTextures         "1"
```
Comment (meag) [inline misc/cfg/gfx_gl_eyecandy.cfg]: 0

File: misc/cfg/gfx_gl_faithful.cfg [hunk 3]
```diff
@@ -156,10 +156,12 @@ gl_colorlights               "0"
 
 //Screen & Powerup Blends
 cl_bonusflash                "0"
+cl_demoplay_flash            "1"
```
Comment (meag) [inline misc/cfg/gfx_gl_faithful.cfg]: 0.33 ?

File: misc/cfg/gfx_gl_faithful.cfg [hunk 4]
```diff
@@ -156,10 +156,12 @@ gl_colorlights               "0"
...
 gl_cshiftpercent             "100"
 gl_hwblend                   "1"
 gl_polyblend                 "0"
-v_contentblend               "1"
+gl_powerupshells             "1"
```
Comment (meag) [inline misc/cfg/gfx_gl_faithful.cfg]: remove gl_powerupshells, or set to 0?  Don't think 'faithful' would have powerup shells

File: misc/cfg/gfx_gl_eyecandy.cfg (hunk 1 above)
Comment (drzel) [reply misc/cfg/gfx_gl_eyecandy.cfg]: No worries. I was getting the defaults from the outdated web manual, I'll lower case these.

File: misc/cfg/gfx_gl_eyecandy.cfg (hunk 2 above)
Comment (drzel) [reply misc/cfg/gfx_gl_eyecandy.cfg]: Yep.

File: misc/cfg/gfx_gl_faithful.cfg (hunk 3 above)
Comment (drzel) [reply misc/cfg/gfx_gl_faithful.cfg]: Ah yeah I guess I thought that faithful would have that sent to one - but actually this PR shouldn't do that so I'll change to default.

File: misc/cfg/gfx_gl_faithful.cfg (hunk 4 above)
Comment (drzel) [reply misc/cfg/gfx_gl_faithful.cfg]: No worries.

File: misc/cfg/gfx_gl_faithful.cfg (hunk 3 above)
Comment (drzel) [reply misc/cfg/gfx_gl_faithful.cfg]: 0.33 is default and same in all so removed.

**Comment by meag** [comment]
Closing - merged

**Pull request closed without merging**